deptrack-devops/src/Devops/Graph.hs

```diff
@@ -148,6 +152,46 @@ type Broadcast = (OpUniqueId,CheckResult,Stability,WantedDirection) -> IO ()
 noBroadcast :: Broadcast
 noBroadcast = const (return ())
 
+-- | Turn-up a graph sequentially
+-- Topologically sort all nodes of the graph then run corresponding operations
+syncTurnupGraph :: Broadcast -> OpGraph -> IO ()
+syncTurnupGraph bcast (graph,lookupVertex,_) =
+  mapM_ go (reverse $ Graph.topSort graph)
```

Comment:
Prelude's implementation of `reverse` is linear AFAIK so complexity of both operations should be equivalent. If you feel more comfortable with using `transposeG` I am fine with that.